pmd-java/src/test/java/net/sourceforge/pmd/lang/java/ast/ASTSwitchStatementTest.java

```diff
@@ -7,14 +7,16 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import net.sourceforge.pmd.lang.java.symboltable.BaseParserTest;
+
 public class ASTSwitchStatementTest extends BaseParserTest {
 
     @Test
     public void exhaustiveEnumSwitchWithDefault() {
-        ASTSwitchStatement switchStatement = getNodes(ASTSwitchStatement.class,
+        ASTSwitchStatement switchStatement = java.parse(
                 "import java.nio.file.AccessMode; class Foo { void bar(AccessMode m) {"
                 + "switch (m) { case READ: break; default: break; } } }")
-                .get(0);
+                .descendants(ASTSwitchStatement.class).firstOrThrow();
```

Comment:
oh... I remember fixing this test class in the opposite direction with the last merge from master....

Reply:
hm... there are some compilation errors in this class....